### DISCUSSION TITLE
Any integration for sso?

### DISCUSSION THREAD

**Discussion by jcklpe** [lukevella/rallly#338] [Ideas]
Biggest single limiting factor I've seen for adopting a new app to an open source web app productivity stack is managing permissions and logins.

I know rally allows users to poll without logging in but if I were to work this into a company work flow I'd probably want it to hook in with the company login solution.

Is there anyway to integrate sso stuff? Has that been done or on the table?

**Comment by lukevella** [comment]
Hi, thanks for the feedback. I do plan on supporting SSO in the future though it's not something I'm currently working on. I'll try and share a roadmap after the next major release to give an idea when this feature can be expected.
